101-traffic-manager-external-endpoint/azuredeploy.parameters.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+    "dnsname": {
+	  "value": "something"
```

Comment:
Please enter the value "#####" so we can test this automatically with the bot. The bot looks for "#####" and replaces with random auto-genned strings